src/lib/pmd/PmdCatalogWrapper.ts

```diff
@@ -2,7 +2,7 @@ import {AnyJson} from '@salesforce/ts-types';
 import fs = require('fs');
 import {Rule} from '../../types';
 import {RuleFilter, RULE_FILTER_TYPE} from '../RuleManager';
-import {PmdSupport, PMD_LIB, PMD_VERSION} from './PmdSupport';
```

Comment:
Don't you need to add PMD_LIB to the start of the PMD standard JAR paths?

Reply:
Yup, burnt my finger. Fixed it.